hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -1,12 +1,29 @@
 /* eslint no-undef: 0 */
 // TODO: Configure eslint to recognize "cy" as a global.
+// We don't click <a> tags here since hrefs redirect to cross-domain urls, which use
+// stubs (to add/remove cookies). HTTP 304 requested urls are not handled by route/request.
 describe('HuBMAP', () => {
   before(() => {
+    Cypress.Cookies.debug(true);
```

Comment:
I would prefer that debugging be turned on only as needed: Otherwise there's clutter in the logs and it makes it harder to debug real bugs. (I have the same concern with all the JS code.)